Business Social Network/AppModules/WishList/views/wishListApp.js

```diff
@@ -1,27 +1,53 @@
 var wishListApp = angular.module('wishListApp', []);
 
+wishListApp.controller('WishListNotificationController',
+  function($scope, $http,$routeParams) {
+    var wishlist_id= '573f4399853cb2ca1affb6dc'; //wishlist_id
+    $scope.getNotifications = function(){   //checks if there are new Notifications meaning if any product
+                                            // in a wishlist has changed
+        var config ={
+        method: "PUT",
+        url: "/wishlists/" + wishlist_id,
+        headers: {"Content-Type": "application/json;charset=utf-8"}
+      };
+              $http(config).then(function(response) {
+
+               if(response.data.flag == '1')    //if data has been updated since last_seen the server controller sets the flag to 1
+               $scope.notificationCount = "New Notification";
+               else
+               $scope.notificationCount = "No Notifications";
+
+
+
+        });
+
+    }
+$scope.getNotifications(); // calling the method
```

Comment:
@HazemElAgaty 
notifications should use a realtime implementation, updates products would not show in user's notification unless the angular app is refreshed